server/route/cbeamRpc.js

```diff
@@ -0,0 +1,19 @@
+require('isomorphic-fetch');
+
+module.exports = (router) => {
+  router.get('/rpc/:method', async (ctx) => {
+    const res = await fetch('http://c-beam.cbrp3.c-base.org:4254/rpc/', {
+      credentials: 'omit',
+      headers: {
+        'content-type': 'application/json',
+      },
+      body: `{"jsonrpc":"2.0","id":1550009052773,"method":"${ctx.params.method}","params":""}`,
```

Comment:
Do we have methods that need parameters?

Reply:
There are some that need parameters, but since infoscreen and the c-beam react frontend both are just "displaying" for now, I didn't add it. We can maybe later add a post route that can send params? Get seems easier for just retrieving data.